### PR TITLE
Documentation for cigetcert

### PR DESCRIPTION
**Pull request by matyasselmeci**

### CODE REVIEW

File: docs/security/user-certs.md [hunk 1]
```diff
@@ -85,12 +89,62 @@ of returning a valid AC for the user.
...
 
 If the security of your certificate or private key has been compromised, you have a responsibility to revoke the certificate. You can follow the steps [here](../security/certificate-management#osg-user-cert-revoke) to revoke your certificate. The same instructions apply if you need to revoke a certificate because your email address changed or your name has changed.
 
+
+Getting a Certificate from a Service Provider with cigetcert
+------------------------------------------------------------
+
+You may also get a user certificate from a SAML 2.0 Service Provider such as your home institution or XSEDE.
+This kind of certificate is short-lived, typically valid only for a week.
+Therefore it is more suitable for running jobs than using in your browser.
```
Comment (brianhlin) [inline docs/security/user-certs.md]: Can we change "running jobs" to "accessing services at OSG sites such as compute elements or storage"?

Comment (matyasselmeci) [reply docs/security/user-certs.md]: That's a mouthful. How about:

Comment (brianhlin) [reply docs/security/user-certs.md]: Sure

**Comment by matyasselmeci** [comment]
Changes made. Could you please re-review?

File: docs/security/user-certs.md [hunk 2]
```diff
@@ -140,10 +140,27 @@ export CIGETCERTOPTS="-i 'Fermi National Accelerator Laboratory'"
 Your VO may also provide specific instructions for how to best use this tool.
 Contact your VO support center for details.
 
-Finally, cigetcert has advanced options for things like selecting an auth method, loading a configuration from a server, or storing the cert on a MyProxy server.
+Finally, `cigetcert` has advanced options for things like selecting an auth method, loading a configuration from a server, or storing the cert on a MyProxy server.
```
Comment (brianhlin) [inline docs/security/user-certs.md]: I didn't comment on this before but let's tighten up the language here and drop `things like`.

File: docs/security/user-certs.md [hunk 3]
```diff
@@ -140,10 +140,27 @@ export CIGETCERTOPTS="-i 'Fermi National Accelerator Laboratory'"
...
 See the [manual page for cigetcert](http://htmlpreview.github.io/?https://github.com/fermitools/cigetcert/blob/master/cigetcert.html) for more information.
 
 
+### Using cigetcert with XSEDE credentials
+
+`cigetcert` also works with XSEDE as the service provider.
+To use XSEDE credentials, you will first need an account at <https://portal.xsede.org>.
+In addition, you _need_ to set up two-factor authentication with XSEDE; see their [MFA documentation](https://portal.xsede.org/mfa) for details.
+Push notifications using the Duo Mobile app are strongly recommended; the login prompt times out too quickly for a phone call to work.
```
Comment (brianhlin) [inline docs/security/user-certs.md]: Let's not even mention the phone call option and say that Duo is a requirement for the cigetcert + xsede method

File: docs/security/user-certs.md [hunk 4]
```diff
@@ -140,10 +140,27 @@ export CIGETCERTOPTS="-i 'Fermi National Accelerator Laboratory'"
...
+
+Once you have set all those up, run `cigetcert` as follows:
+```console
+user@host $ cigetcert -u %RED%<USERNAME>%ENDCOLOR% -i XSEDE
+```
+`<USERNAME>` is your username at portal.xsede.org.
+You will get prompted to "Enter XSEDE Kerberos Password."
+Enter the password for your account at portal.xsede.org.
+You should then get a 2FA authentication request using the method you set up above.
+Once you complete the request (for example, accept the Duo Mobile notification), `cigetcert` will issue the certificate.
```
Comment (brianhlin) [inline docs/security/user-certs.md]: Since I'd like to see us just talk about Duo, let's merge these two sentences and only talk about Duo

**Pull request merged**